reports/RJ722-II.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/RahulJha
+student: Rahul Jha
+organisation: coala
+organisation_link : https://coala.io
+project: Improve Whitelisting Infrastructure for Vulture
+project_link: https://summerofcode.withgoogle.com/projects/#6564412442804224
+report_year: 2018
+tarball: https://drive.google.com/file/d/12HgBb2Wdjl6Zzm7mDqCt13ke8VHj7_Bb/view
+mentors: >
+ [Jendrik Seipp](https://github.com/jendrikseipp) & [Abdeali Kothari](https://github.com/AbdealiJK) & [Florian Bruhin](https://github.com/The-Compiler)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/29
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/30
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/31
+bio: >
+ Rahul is currently a junior, studying Electronics Engineering at Zakir Hussain
+ College of Engineering and Technology, Aligarh Muslim University. He loves the
+ ideology behind Free and Open Source Software and reads about the history of
+ this revolutionary movement in his leisure time.
+social:
+ - GitHub:
+   - username: RJ722
+   - link: https://github.com/RJ722
+ - GitLab:
+   - username: RJ722
+   - link: https://gitlab.com/RJ722
+ - Gitter:
```

Comment:
Add OpenHub

Reply:
You should create a project for https://github.com/RJ722/vulture-whitelist-generators at https://www.openhub.net/p/new

https://www.openhub.net/accounts/RJ722/positions

Reply:
Still not done

Reply:
Added the openhub profile link. Creating the project on openhub.

Reply:
Here it is: https://www.openhub.net/p/vulture-whitelist-generators